File: Python/1018 - Cédulas.py
# author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
# copyright: Josélio Júnior (Lunatic Fox) 2022

import math

v = int(input())
a, n = [str(v)], [100, 50, 20, 10, 5, 2, 1]

for i in range(len(n)):
  a.append(f'{str(math.floor(v / n[i]))} nota(s) de R$ {str(n[i])},00')
  v %= n[i]

print('\n'.join(a))

File: JavaScript/1018 - Cédulas.js
/**
 * @author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
 * @copyright Josélio Júnior (Lunatic Fox) 2021
*/

const p = require('fs').readFileSync('/dev/stdin', 'utf8'),
    l = p.split('\n');

let v = +l[0];

const a = [v],
    n = [100, 50, 20, 10, 5, 2, 1];

for (let i = 0; i < n.length; i++) {
    a.push(`${parseInt(v / n[i])} nota(s) de R$ ${n[i]},00`);
    v %= n[i];
}

console.log(a.join('\n'));